recipes/reago/meta.yaml

```diff
@@ -27,5 +27,5 @@ test:
 
 about:
   home: https://github.com/chengyuan/reago-1.1
```

Comment:
You should add more tests to be sure it is working.
The CM files are not copied over isn't it?